lib/chapters_manager.php

```diff
@@ -82,6 +82,12 @@ private function get_raw_chapters_string() {
 	private function get_chapters_object() {
 
 		if ( ! $this->chapters_raw )
+			$this->chapters_raw = apply_filters( 'podlove_episode_chapters', $this->episode->post_id );
```

Comment:
Wenn der Filter nicht benutzt wird, ist `$this->chapters_raw = $this->episode->post_id`.

```
apply_filters( 'podlove_episode_chapters', NULL, $this->episode->post_id );
```

sollte funktionieren.